lib/warder/runner.rb

```diff
@@ -32,13 +42,24 @@ def command_with_options
     end
 
     def failed?(line)
-      klass = self.class
       match = klass::FAILURE_REGEXP.match(line)
       match && match[1].to_i != klass::FAILURE_THRESHOLD
     end
 
-    def printable?(*)
-      true
+    def printable?(_line)
+      !quiet?
+    end
+
+    def quiet?
+      @options['quiet']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.